engines/dfc_provider/app/services/quantitative_value_builder.rb

```diff
@@ -33,7 +33,7 @@ def self.apply(quantity, product)
     product.variant_unit = measure
     product.variant_unit_name = unit_name if measure == "items"
     product.variant_unit_scale = unit_scale
-    product.unit_value = quantity.value * unit_scale
+    product.unit_value = quantity.value.to_f * unit_scale
```

Comment:
> This should probably be handled in the Connector long-term.

Good decision. I can't think of any downside to being tolerant of strings here. It may save time with other not-quite-right integrations.

Reply:
I opened an issue for this:

* https://github.com/datafoodconsortium/connector-ruby/issues/21